main/src/ca/uwaterloo/flix/runtime/interpreter/Channel.java

```diff
@@ -71,7 +71,7 @@ public Channel(int bufferSize) {
 
   public static void spawn(Spawnable s) {
     // Create a new Thread and evaluate the spawned expression in the new Thread
-    Thread thread = new Thread(s::spawn);
+    Thread thread = new Thread(s::spawn); // Q: How does this compile given that 'Spawnable' doesn't implement 'Runnable'
```

Comment:
This is actually using some newish Java syntax/magic where `s::spawn` grabs the `spawn` method of the `s` object and turns it into a Runnable. 

https://dzone.com/articles/java-lambda-method-reference

So the comment can be removed :)